### PR TITLE
address various lints in src/io/flutter/perf, src/io/flutter/performance, src/io/flutter/module

### PR DESCRIPTION
**Pull request by devoncarew**
- address various lints in src/io/flutter/perf, src/io/flutter/performance, src/io/flutter/module

### CODE REVIEW

File: src/io/flutter/performance/FrameRenderingDisplay.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     df.setMaximumFractionDigits(1);
   }
 
-  public static JPanel createJPanelView(Disposable parentDisposable, FlutterApp app) {
+  public static JPanel createJPanelView(Disposable parentDisposable, @NotNull FlutterApp app) {
```
Comment (stevemessick) [inline src/io/flutter/performance/FrameRenderingDisplay.java:38]: Both args should be `@NotNull`.

File: src/io/flutter/module/FlutterModuleBuilder.java [hunk 2]
```diff
@@ -284,7 +284,7 @@ public String getBuilderId() {
 
   @Override
   @NotNull
-  public ModuleType getModuleType() {
+  public ModuleType<?> getModuleType() {
```
Comment (stevemessick) [inline src/io/flutter/module/FlutterModuleBuilder.java:287]: Line 205 needs `Nullable` on the first parameter.

**Review by stevemessick — APPROVED** [review]
LGTM with a couple nits.

File: src/io/flutter/module/FlutterModuleBuilder.java (hunk 2 above)
Comment (devoncarew) [reply src/io/flutter/module/FlutterModuleBuilder.java:287]: done

File: src/io/flutter/performance/FrameRenderingDisplay.java (hunk 1 above)
Comment (devoncarew) [reply src/io/flutter/performance/FrameRenderingDisplay.java]: done

**Pull request merged**